rfd/0228-requesting-specific-principals-for-resources.md

```diff
@@ -0,0 +1,117 @@
+| Authors                                  | State |
+|------------------------------------------|-------|
+| Maxim Dietz (maxim.dietz@goteleport.com) | draft |
+
+# RFD 0228 - Requesting Specific Principals for Resources
+
+## Required Approvers
+- Engineering: (@r0mant && @smallinsky)
+
+## Related
+- See [#58307](https://github.com/gravitational/teleport/issues/58307)
```

Comment:
fyi:
- access request to db-name: https://github.com/gravitational/teleport/issues/59486
- access request to db-role: https://github.com/gravitational/customer-sensitive-requests/issues/468